app/scripts/controllers/network/network-controller.ts

```diff
@@ -1148,4 +1148,14 @@ export class NetworkController extends EventEmitter {
       networkConfigurations,
     });
   }
+
+  getNetworkConfigurationForChainId(chainId: string) {
```

Comment:
should split this off into seperate PR, since it will need to go into core/network-controller as well.